test/colony.js

```diff
@@ -614,15 +657,6 @@ contract("Colony", accounts => {
     it("should allow manager to assign himself to a role", async () => {
       const taskId = await makeTask({ colony });
 
-      await executeSignedRoleAssignment({
-        colony,
-        taskId,
-        functionName: "setTaskEvaluatorRole",
-        signers: [MANAGER],
-        sigTypes: [0],
-        args: [taskId, MANAGER]
-      });
```

Comment:
We can remove the unnecessary assert that checks this too further down in test, i.e. 
```
const evaluatorInfo = await colony.getTaskRole(taskId, EVALUATOR_ROLE);
assert.equal(evaluatorInfo[0], MANAGER);
```

Reply:
Yup